scripts/SMCodeGen/dataModels/stateMachineAppModel.json

```diff
@@ -268,7 +268,15 @@
                                     "ABORT_ATTACH"
                                 ],
                                 "NextState":"end_state"
-                             }
+                             },
+                             {
+                                "Name":"STATE_GUARD_TIMEOUT",
+                                "Actions":[
+                                    "HANDLE_STATE_GUARD_TIMEOUTS_FOR_CSREQ_IND",
+                                    "ABORT_CSREQ_INDICATION"
```

Comment:
ABORT_CSREQ_INDICATION can be removed?

Reply:
Opps... Yes. Removed it from auto generated code and forgot to remove from here.